apps/fluent-tester/src/TestComponents/Theme/ThemeTest.android.tsx

```diff
@@ -0,0 +1,67 @@
+import * as React from 'react';
+import { Theme, useTheme } from '@fluentui-react-native/theme-types';
+import { themedStyleSheet } from '@fluentui-react-native/themed-stylesheet';
+import { THEME_TESTPAGE } from '../../../../E2E/src/Theme/consts';
+import { Test, TestSection, PlatformStatus } from '../Test';
+import { Text as TextRN, View } from 'react-native';
+import { Text } from '@fluentui/react-native';
+
+import { useFluentTheme } from '@fluentui-react-native/framework';
+
+const getThemedStyles = themedStyleSheet((theme: Theme) => {
+  return {
+    swatch: {
+      color: theme.colors.neutralForeground1,
+    },
+    rootView: {
+      backgroundColor: theme.colors.brandBackground,
+    },
+  };
+});
+
+const Panel: React.FunctionComponent = () => {
+  const theme = useFluentTheme();
+  const theme2 = useTheme();
```

Comment:
useFluentTheme should have everything on useTheme. the difference useFluentTheme has is that it returns a fallback theme if the ThemeProvider isn't used.

Reply:
Ok, thanks for clarifying. I was initially thinking to demonstrate usage of both as doing the same thing - but I think the guidance would be to use `useFluentTheme`() - so kept only that.

Reply:
I am not also aware if there is another place that has a similar example on `ThemeRecipes` usage - if there is another one that does similar/ I should extend this example to other endpoints, let me know and I can make the changes.

Reply:
I usually point people to the win32 theme (the impl of createOfficeTheme) as an example of using ThemeRecipes, but other than that I don't think we have an example of it.